README.md

```diff
@@ -81,4 +82,36 @@ You can change any of these by modifying `actions.py` and the corresponding NLU
 ## Testing the bot
 
 You can test the bot on the test conversations by running  `rasa test`. 
-Note that if duckling is running when you do this, you'll probably see some "failures" because of entities; that's ok! 
\ No newline at end of file
+This will run [end-to-end testing](https://rasa.com/docs/rasa/user-guide/testing-your-assistant/#end-to-end-testing) on the conversations in `tests/conversation_tests.md`. 
+
+Note that if duckling is running when you do this, you'll probably see some "failures" because of entities; that's ok! Since duckling entities' extraction is not influenced by NLU training data, and since `time` entities' values depend on when the tests are being run, these have been left unannotated in the conversation tests.
```

Comment:
```suggestion
Note that if duckling is running when you do this, you'll probably see some "failures" because of entities; that's ok! Since duckling entity extraction is not influenced by NLU training data, and since the values of time entities depend on when the tests are being run, these have been left unannotated in the conversation tests.
```